opencti-platform/opencti-graphql/package.json

```diff
@@ -19,7 +19,11 @@
     "start-cluster": "NODE_ENV=cluster node build/back.js",
     "serv": "node build/back.js",
     "insert:dev": "node build/script-insert-dataset.js",
-    "test:dev": "vitest watch --config vitest.config.dev.ts",
+    "test:dev:init": "INIT_TEST_PLATFORM=true vitest run --config vitest.config.dev.ts --testNamePattern=\"Should import creation succeed\" tests/02-integration/00-inject/loader-test.ts",
```

Comment:
This step is not working on my computer, "test:dev:init" is not working on my computer, I have a error at "Should import creation succeed" step,

Reply:
I suppose you do not have more details ?
Are you sure it's not related to a python client version mismatch on your local setup?

Reply:
ok testing again now